src/App.tsx

```diff
@@ -20,32 +20,36 @@ import { createTheme, ThemeProvider } from '@mui/material/styles'
 
 const theme = createTheme({
   typography: {
-    fontFamily: 'Lato, sans-serif',
+    fontFamily: 'Roboto, sans-serif',
     fontSize: 16,
-    fontWeightRegular: 300,
-    fontWeightBold: 400,
+    fontWeightRegular: 400,
+    fontWeightBold: 500,
     h1: {
-      fontWeight: 400,
+      fontWeight: 500,
     },
     h2: {
-      fontWeight: 400,
+      fontWeight: 500,
     },
     h3: {
       fontSize: '3rem',
-      fontWeight: 400,
+      fontWeight: 500,
     },
     h4: {
       fontSize: '2rem',
-      fontWeight: 400,
+      fontWeight: 700,
     },
     h5: {
       fontSize: '1.2rem',
-      fontWeight: 400,
+      fontWeight: 700,
     },
     body1: {
       fontSize: '1.25rem',
       lineHeight: 1.5,
     },
+    button: {
+      textTransform: 'none',
+      fontWeight: 500,
+    },
```

Comment:
since we're using the default roboto and maybe some default fontweights, let's make sure we only specify the properties that aren't already the default. i.e. no redundancies

Reply:
also why are the fontWeights different for h4 and h5?

Reply:
cause their fontsize is small, having them more bold has more emphasis, whereas h1, h2, h3 have bigger fontsize and having lesser bold looks good.

I have different fontweight for each header now, cause there is at least one use of every header except h6, and the headings use is not followed properly as each page is using them differently. I plan to go through the whole website and edit tags/css properly, that why the pages stay consistent.